packages/apsara-ui/src/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { StyledSkeleton, ListItem, StyledRow } from './Skeleton.styles';
+
+interface ListSkeletonProps {
+    className?: string;
+    rows?: number;
+    width?: string;
+}
+export function ListSkeleton({ className = "", rows = 20, width = "100%" }: ListSkeletonProps) {
+    return (
+        <StyledSkeleton className={className} $props={{ "width": width }}>
+            <ul>
+                {[...Array(rows)].map(() => <ListItem />)}
+            </ul>
+        </StyledSkeleton>
+    )
+}
+
+interface EditorSkeletonProps {
+    className?: string;
+    width?: string;
+    lastLineWidth?: string
+}
+export function EditorSkeleton({ className = "", width = "100%", lastLineWidth = "100%" }: EditorSkeletonProps) {
```

Comment:
wdyt about `Form*` instead of `Editor*`? Will that improve it?

Reply:
yeah we can definitely rename it, I just kept the previously used names.